bundles/org.openhab.ui.homebuilder/web/dist/build.js

```diff
@@ -0,0 +1,37 @@
||"<Nothing selected>"))]),t._l(t.items,function(e){return n("option",{domProps:{value:t.getItemID(e)}},[t._v(t._s(t.getItemName(e)))])})],2)},staticRenderFns:[]}},function(t,e,n){n(221);var r=n(3)(n(222),n(223),null,null);t.exports=r.exports},2,function(t,e,n){"use strict";function r(t){return t&&t.__esModule?t:{default:t}}Object.defineProperty(e,"__esModule",{value:!0});var i=n(15),o=r(i),a=n(147),l=r(a);e.default={mixins:[l.default],methods:{click:function(){(!0!==this.schema.validateBeforeSubmit||this.$parent.validate())&&(0,o.default)(this.schema.onSubmit)&&this.schema.onSubmit(this.model,this.schema)}}}},function(t,e){t.exports={render:function(){var t=this,e=t.$createElement;return(t._self._c||e)("input",{attrs:{type:"submit",name:t.schema.inputName,disabled:t.disabled},domProps:{value:t.schema.buttonText},on:{click:t.click}})},staticRenderFns:[]}},function(t,e,n){n(225);var r=n(3)(n(226),n(227),null,null);t.exports=r.exports},2,146,function(t,e){t.exports={render:function(){var t=this,e=t.$createElement;return(t._self._c||e)("textarea",{directives:[{name:"model",rawName:"v-model",value:t.value,expression:"value"}],staticClass:"form-control",attrs:{id:t.getFieldID(t.schema),disabled:t.disabled,maxlength:t.schema.max,minlength:t.schema.min,placeholder:t.schema.placeholder,readonly:t.schema.readonly,rows:t.schema.rows||2,name:t.schema.inputName},domProps:{value:t._s(t.value)},on:{input:function(e){e.target.composing||(t.value=e.target.value)}}})},staticRenderFns:[]}},function(t,e,n){function r(t){return n(i(t))}function i(t){return o[t]||function(){throw new Error("Cannot find module '"+t+"'.")}()}var o={"./fieldCleave.vue":229,"./fieldDateTimePicker.vue":233,"./fieldGoogleAddress.vue":245,"./fieldImage.vue":249,"./fieldMasked.vue":253,"./fieldNoUiSlider.vue":257,"./fieldPikaday.vue":261,"./fieldRangeSlider.vue":265,"./fieldSelectEx.vue":295,"./fieldSpectrum.vue":299,"./fieldStaticMap.vue":303,"./fieldSwitch.vue":307,"./fieldVueMultiSelect.vue":311};r.keys=function(){return Object.keys(o)},r.resolve=i,t.exports=r,r.id=228},function(t,e,n){n(230);var r=n(3)(n(231),n(232),null,null);t.exports=r.exports},2,function(t,e,n){"use strict";function r(t){return t&&t.__esModule?t:{default:t}}Object.defineProperty(e,"__esModule",{value:!0});var i=n(185),o=r(i),a=n(147),l=r(a);e.default={mixins:[l.default],data:function(){return{cleave:null}},mounted:function(){this.$nextTick(function(){window.Cleave?this.cleave=new window.Cleave(this.$el,(0,o.default)(this.schema.cleaveOptions||{},{creditCard:!1,phone:!1,phoneRegionCode:"AU",date:!1,datePattern:["d","m","Y"],numeral:!1,numeralThousandsGroupStyle:"thousand",numeralDecimalScale:2,numeralDecimalMark:".",blocks:[],delimiter:" ",prefix:null,numericOnly:!1,uppercase:!1,lowercase:!1})):console.warn("Cleave is missing. Please download from https://github.com/nosir/cleave.js/ and load the script in the HTML head section!")})},beforeDestroy:function(){this.cleave&&this.cleave.destroy()}}},function(t,e){t.exports
```

Comment:
Are the files in "web/dist" meant to be checked in? I thought they were created during build time?

Reply:
For now they are, but I plan to chain it with Maven like @resetnow did with BasicUI in  https://github.com/eclipse/smarthome/pull/4040